nursery/capture-webcam-video.yml

```diff
@@ -0,0 +1,25 @@
+rule:
+  meta:
+    name: capture webcam video
+    namespace: collection/webcam
+    author: "@johnk3r"
+    description: Rule that detects a system's webcam being used to capture video
+    scope: function
+    att&ck:
+      - Collection::Video Capture [T1125]
+  features:
+    - or:
+      - and:
+        - api: capCreateCaptureWindow
+        - basic block:
+          - and:
+            - api: SendMessage
+            - number: 0x43E = WM_CAP_SEQUENCE
```

Comment:
thanks for noticing that `capCaptureSequence` is a macro and gets compiled to `SendMessage/WM_CAP_SEQUENCE`